src/components/item/sharing/ItemSharingTab.tsx

```diff
@@ -33,6 +34,27 @@ type Props = {
   memberships?: ItemMembership[];
 };
 
+interface PermissionMap {
+  [key: string]: ItemMembership;
+}
+
+const reduceMembershipsForSameMember = (
```

Comment:
Nitpicking: I would change the name of this function to something like: `selectHighestMemberships`. The `ForSameMember` feels a bit weird. It should read naturally when you use it in a component prop.